docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+
+  - locale
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.HourFormat.EventName`           | `SystemEvents.HourFormat.Key`             | - `SystemEvents.HourFormat.Value12` <br>- `SystemEvents.HourFormat.Value24`                             | When  the `hour_format` has changed,  for example, by the user toggling the date and time settings for the 24-hour  clock (where **OFF** stands  for the 12-hour clock). | You  can get the current value with the `System.SystemSettings.LocaleTimeFormat24HourEnabled` property. |
+    | `SystemEvents.LanguageSet.EventName`          | `SystemEvents.LanguageSet.Key`           | The value of  this key is the full name of the locale, for example, `ko_KR.UTF8` for Korean and `en_US.UTF8` for American English. For more information, see the Linux  locale information. | When the `language_set` has changed.       | You can get the  current value with the `System.SystemSettings.LocaleLanguage` property. |
+    | `SystemEvents.RegionFormat.EventName`         | `SystemEvents.RegionFormat.Key`           | The  value of this key is the full name of the locale, for example, `ko_KR.UTF8` for the Korean region  format and `en_US.UTF8`  for the USA region format. For more information, see the Linux locale  information. | When  the `region_format` has changed.     | You  can get the current value with the `System.SystemSettings.LocaleCountry` property. |
+
+  - sound
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.SilentMode.EventName`           | `SystemEvents.SilentMode.Key`             | - `SystemEvents.SilentMode.ValueOn` <br> - `SystemEvents.SilentMode.ValueOff`                            | When  the ringtone has been changed to 0 or another mode. For example, if the call  slider has been changed to 0, `silent_mode` is `"on"`. Otherwise, `silent_mode` is `"off"`. | You  can get the current value with the `System.SystemSettings.SoundSilentModeEnabled` property. |
+
+  - vibration
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.VibrationState.EventName`       | `SystemEvents.VibrationState.StateKey`          | - `SystemEvents.VibrationState.StateValueOn` <br> - `SystemEvents.VibrationState.StateValueOff`                           | When the  vibration state has changed.   | You can get the current value using System.Information.SetCallback with the `http://tizen.org/runtimefeature/vibration` property. |
```

Comment:
How about `System.Information.SetCallback`? Applying font style to System.Information.SetCallback.